### ISSUE TITLE
If a local IPFS node is running before trying to start ipfs-desktop,  ipfs-desktop throws an unhandled error

### ISSUE THREAD

**Issue by thorsten-panknin** [ipfs-shipyard/ipfs-desktop#653]
Hi,

I'm on an updated Manjaro Linux, 64-Bit, and I have my own local ipfs node running fine.

I was able to install ipfs-desktop via "npm install", but running "npm start" opens up a dialog explaining that there was an unhandled error and that I should open an issue with the info from the error logs. Problem is, there are no error logs, I only have the npm error logs. Please see below.

I think I narrowed it down: the local node cannot be running already, or a "unhandled error" will be thrown. After I stopped the IPFS node, ipfs-desktop started up fine. I had a look through error.js which gave me the idea the cause might be a running node. It seems that the error handling doesn't work there though. (This might sound like I've got more than the most passing idea of what I'm talking about, I just clued things together though)

```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'start' ]
2 info using npm@6.4.0
3 info using node@v8.11.4
4 verbose run-script [ 'prestart', 'start', 'poststart' ]
5 info lifecycle ipfs-desktop@0.5.0~prestart: ipfs-desktop@0.5.0
6 info lifecycle ipfs-desktop@0.5.0~start: ipfs-desktop@0.5.0
7 verbose lifecycle ipfs-desktop@0.5.0~start: unsafe-perm in lifecycle true
8 verbose lifecycle ipfs-desktop@0.5.0~start: PATH: /usr/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/xyz/vcs/ipfs-desktop/node_modules/.bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/home/xyz/.local/share/flatpak/exports/bin:/var/lib/flatpak/exports/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl
9 verbose lifecycle ipfs-desktop@0.5.0~start: CWD: /home/xyz/vcs/ipfs-desktop
10 silly lifecycle ipfs-desktop@0.5.0~start: Args: [ '-c', 'electron-forge start' ]
11 silly lifecycle ipfs-desktop@0.5.0~start: Returned: code: 1  signal: null
12 info lifecycle ipfs-desktop@0.5.0~start: Failed to exec start script
13 verbose stack Error: ipfs-desktop@0.5.0 start: `electron-forge start`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:304:16)
13 verbose stack     at emitTwo (events.js:126:13)
13 verbose stack     at EventEmitter.emit (events.js:214:7)
13 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at emitTwo (events.js:126:13)
13 verbose stack     at ChildProcess.emit (events.js:214:7)
13 verbose stack     at maybeClose (internal/child_process.js:925:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
14 verbose pkgid ipfs-desktop@0.5.0
15 verbose cwd /home/xyz/vcs/ipfs-desktop
16 verbose Linux 4.18.3-1-MANJARO
17 verbose argv "/usr/bin/node" "/usr/bin/npm" "start"
18 verbose node v8.11.4
19 verbose npm  v6.4.0
20 error code ELIFECYCLE
21 error errno 1
22 error ipfs-desktop@0.5.0 start: `electron-forge start`
22 error Exit status 1
23 error Failed at the ipfs-desktop@0.5.0 start script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

**Comment by hacdias** [comment]
Hello @thorsten-panknin! Sorry for this, it will hopefully be solved with #651.

**Comment by hacdias** [comment]
Closing in favour of #689 and #688
